src/PowerShellEditorServices/Services/PowerShell/Host/PsesInternalHost.cs

```diff
@@ -532,14 +539,7 @@ private void RunTopLevelExecutionLoop()
                 // Signal that we are ready for outside services to use
                 _started.TrySetResult(true);
 
-                if (_hostInfo.ConsoleReplEnabled)
-                {
-                    RunExecutionLoop();
-                }
-                else
-                {
-                    RunNoPromptExecutionLoop();
-                }
```

Comment:
And also around this!

Reply:
@dkattan I'm removing this `RunNoPromptExecutionLoop()` FYI, but like `RunExecutionLoop` works without an integrated console just fine based on a) its logic that handles no console being available and b) these unit tests that work without a console using it.